bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -108,7 +108,7 @@
  * @author Chris Jackson
  * @author Yordan Zhelev - Added Swagger annotations
  */
-@Component(service = { SitemapResource.class, RESTResource.class })
+@Component
```

Comment:
This will expose the service with all attached interfaces. Please use `@Component(service = RESTResource.class )` here.